### ISSUE TITLE
[Question]: Parsed fist page

### ISSUE THREAD

**Issue by Domi31tls** [infiniflow/ragflow#1740]
### Describe your problem

Hello everyone !
Could you help me: After integrating my documents, I realize that some pages are not integrated, especially the first ones? Additionally, the structure of the documents is similar. How can I ensure that the system reliably detects the data?

**Comment by KevinHuSh** [comment]
Is the first page about content table  or something useless?
It will remove content table/footer/header automatically.

**Comment by Domi31tls** [comment]
No, the documents are deeds drafted by notaries. They all have the same structure. They can be around ten pages long. I encounter this problem mainly with complete PDFs. If I truncate the document to keep the first 2 pages and convert it to text to create the chunk, I have fewer problems. I am attaching a copy of the structure as a sample. I have highlighted the important information.
![ragflow](https://github.com/user-attachments/assets/3d15044e-ac21-4f87-9a9b-05aa39014de1)

Thank for your help
